.github/workflows/check.yml

```diff
@@ -4,10 +4,24 @@ on: [push, pull_request]
 
 jobs:
   check:
-    name: run checks
+    strategy:
+      fail-fast: false
+      matrix:
+        tox-filter:
+          - "py3.8"
+          - "py3.9"
+          - "py3.10"
+          - "py3.11"
+          - "py3.12"
+          - "py3.13"
```

Comment:
I like this idea, however it solve issue partially as we still has different django, psycopg and postgres versions. Is it possible to move tox matrix that include python, django, psycopg, postgres versions fully to github actions matrix?

Reply:
Yes it's true. It's a trade-off between CI workflow configuration complexity and convenience. If the split is fine grained down to 1 CI job per tox env, then your logs are super clear, but the YAML can get quite complicated. 

I find that having 1 job per Python version is usually a good balance... I know of a few Django packages I use doing that (whitenoise, django-cors-headers) and that's the approach in the @django-commons org as well.

Reply:
ok, your approach anyway better that current